src/components/expandable-row.vue

```diff
@@ -14,18 +14,19 @@ except according to the terms contained in the LICENSE file.
     <div class="title-cell">
       <slot name="title"></slot>
     </div>
-    <!-- eslint-disable-next-line vuejs-accessibility/click-events-have-key-events, vuejs-accessibility/interactive-supports-focus -->
-    <div role="button" class="caption-cell" @click.prevent="toggleExpanded">
+    <!-- eslint-disable-next-line vuejs-accessibility/click-events-have-key-events -->
+    <div class="caption-cell" @click="toggleExpanded">
```

Comment:
I added a similar click handler to `.button-cell` below. I thought I'd try to make the two divs similar, so I removed `.prevent` here, which I thought probably wasn't needed. I also removed `role="button"`, since users using a screen reader will use the button itself. I don't think we need to announce the div as a button. I want it to be clear though to users using a mouse that they can click, so I added `cursor: pointer;` to the CSS below.